mc/query/query.peg.go

```diff
@@ -0,0 +1,2108 @@
+package query
```

Comment:
hmm is the convention in golang land to keep everything checked in? I generally like to ignore all generated files if the build tool is fast, but I guess it's up to you since you are the one spending the most time in here 😄